_example/echo.proto

```diff
@@ -10,7 +10,7 @@ package echo;
 
 import "github.com/TuneLab/go-genproto/googleapis/api/serviceconfig/annotations.proto";
 
-service Echo {
+service Bounce_Echo {
```

Comment:
That was a mistake, which I thought I fixed, that snuck in from debugging. Whoops!